examples_deepspeed/universal_checkpointing/tb_analysis/tb_analysis_script.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+import os
+import re
+import pandas as pd
+import matplotlib.pyplot as plt
+from tensorboard.backend.event_processing.event_accumulator import EventAccumulator
+from utils import get_analyzer, find_files_recursive
+from arguments import parser
+
+args = parser.parse_args()
+
+if args.use_sns:
+    try:
+        import seaborn as sns
+        sns.set()
+    except ImportError as e:
```

Comment:
If a user provides `--use_sns` I think we should error out with the package is not found.

Reply:
Addressed, thanks.